ogc-rs/src/system.rs

```diff
@@ -1,6 +1,6 @@
-//! The ``system`` module of ``ogc-rs``.
+//! The ``pad`` module of ``ogc-rs``.
 //!
-//! This module implements a safe wrapper around the OS functions found in ``system.h``.
+//! This module implements a safe wrapper around the gamecube controller functions found in ``pad.h``.
```

Comment:
The documentation here seems to be for the wrong file, might have just slipped up in reverting.

Reply:
Ops.